device-types/Supermicro/SSE-X3348TR.yaml

```diff
@@ -0,0 +1,124 @@
+---
+manufacturer: Supermicro
+model: Supermicro SSE-X3348TR
+slug: supermicro-sse-x3348tr
+part_number: SSE-X3348TR
+u_height: 1
+is_full_depth: true
+comments: '[SSE-X3348T_Switch_Installation_Guide.pdf](https://www.supermicro.com/manuals/network/SSE-X3348T_Switch_Installation_Guide.pdf)'
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+  - name: PS2
+    type: iec-60320-c14
```

Comment:
Hi, Thank you for contribution.

As this switch has hot swappable PSUs they should be modeled under `module-bays` and not `power-ports` then add the required PSU modules under `module-types/Supermicro/` if they don't already exist.